library/src/main/java/com/novoda/downloadmanager/Download.java

```diff
@@ -3,11 +3,17 @@
 public class Download {
 
     private final long id;
+    private final String title;
+    private final String description;
+    private final int status;
     private final long currentSize;
     private final long totalSize;
 
-    public Download(long id, long currentSize, long totalSize) {
+    public Download(long id, String title, String description, int status, long currentSize, long totalSize) {
```

Comment:
why do we need to pass the status? The customiser knows what status the download is right?